### PR TITLE
CMake fixes

### PR DESCRIPTION
**Pull request by Spixmaster**
Set needed variable.
Fixed https://github.com/unum-cloud/usearch/issues/522.

### CODE REVIEW

**Comment by Spixmaster** [comment]
These fixed are really needed. Please merge and publish a new version.

**Comment by ashvardanian** [comment]
@Spixmaster can look into this later today, thanks!

File: CMakeLists.txt [hunk 1]
```diff
@@ -357,3 +346,30 @@ if(${STRINGZILLA_BUILD_SHARED})
 
 
 endif()
+
+if(STRINGZILLA_INSTALL)
+  install(
+    TARGETS stringzilla_shared
+    ARCHIVE
+    BUNDLE CXX_MODULES_BMI
```
Comment (ashvardanian) [inline CMakeLists.txt:354]: We don't need CXX BMI here as well, I believe.

Comment (Spixmaster) [reply CMakeLists.txt:354]: I understand for SimSIMD as it is written in C but StringZilla is a C++ library. I would install everything that is related to C++ for a C++ library.

Comment (ashvardanian) [reply CMakeLists.txt:354]: Not really, StringZilla in-core is two C99 libraries: header-only and pre-compiled shared lib with dynamic dispatch. There is also a third header-only C++11 wrapper, so we need to install them all with different settings. But this requires a full CMake rewrite and will take much more time.

It's expected with the next major release.

Comment (Spixmaster) [reply CMakeLists.txt:354]: Yes the CMake files really need a revamp. I would suggestall projects SimSIMD, StringZilla and usearch.

Comment (Spixmaster) [reply CMakeLists.txt:354]: Stille remove `CXX_MODULES_BMI`?

Comment (ashvardanian) [reply CMakeLists.txt:354]: Yes, let's remove the BMI for now, and keep the bigger changes for future. If you have broader changes in mind, can you please comment on this [PR](https://github.com/ashvardanian/StringZilla/pull/85)?

**Pull request merged**